File: programmers/lev1/12950.py
def solution(arr1, arr2):
    answer = [[0]*len(arr1[0]) for _ in range(len(arr1))]
    for i in range(len(arr1)):
        for j in range(len(arr1[0])):
            answer[i][j]=arr1[i][j]+arr2[i][j]
        
    print(answer)
    return answer

arr1=[[1,2],[2,3]]
arr2=[[3,4],[5,6]]
solution(arr1,arr2)

File: java/src/programmers/year2023/lv1/_12950.java
package programmers.year2023.lv1;

public class _12950 {
    public static void main(String[] args) {
        int[][] arr1 = {{1,2},{2,3}};
        int[][] arr2 = {{3,4},{5,6}};

        solution(arr1, arr2);
    }

    public static int[][] solution(int[][] arr1, int[][] arr2) {
        for(int i=0 ; i<arr1.length ; i++) {
            for(int j=0 ; j<arr1[0].length ; j++) {
                arr1[i][j] += arr2[i][j];
            }
        }
        return arr1;
    }
}
